docs/development/decision-records/2023-02-09-release-process/README.md

```diff
@@ -0,0 +1,117 @@
+  when moving to OSS.
+- adhere to GitFlow branch naming conventions, i.e. no `docu`, `tryout_something` branches. Increases readability and
+  clarity, improves tool support.
+
+_Other guidelines w.r.t. the review process, merging etc. will follow in a later DR._
+
+## Further considerations
+
+### A word on Bugfixes/Hotfixes
+
+Once a release is published, for example `0.3.0` it will receive no further development other than hotfixes. Similarly,
+hotfix branches are created based off of the release branch, here `releases/0.3.0`, thus, `hotfix/0.3.1`. From this,
+three scenarios emerge:
+
+1. The actual fix is done on `develop` and can be cherry-picked into the `hotfix/0.3.1` branch. No new commits are
+   made directly in that branch.
+2. The actual fix is done on `develop` and must be manually ported into the `hotfix/0.3.1` branch. One or several new
+   commits are made on `hotfix/0.3.1`. This is needed when cherry-picking is not available due to incompatibilities
+   between `develop` and the hotfix branch due to intermittent changes.
+3. The fix is only relevant for the `0.3.1` hotfix, it is not needed in `develop`. This can happen, when the problem is
+   not present on `develop`, because it was already implicitly fixed, or otherwise doesn't exist.
+
+This might produce many branches, and the first `hotfix` makes the release obsolete, but it will greatly help
+readability and make a release's history readily apparent.
+
+### Nightly builds
+
+Nightly builds are generated according to a fixed schedule (as suggested
+in [this Jira issue](https://jira.catena-x.net/browse/A1IDSC-408)). Upstream EDC will soon begin to publish nightly
+builds as actual releases (as opposed to: snapshots) to a separate OSSRH-operated repository (see
+[this EDC decision record](https://github.com/eclipse-edc/Connector/tree/main/docs/developer/decision-records/2023-02-10-nightly-builds)).
+
+Unfortunately there is no way to automatically trigger the tractusx-edc build whenever a new EDC nightly is
+created. The most reliable method is to periodically query for the latest EDC nightly, e.g. leveraging GitHub's `dependabot` feature, or using the method highlighted in the aforementioned Jira issue:
+
+```shell
+curl <OSSHR_RELEASES_REPO_URL>/org/eclipse/edc/connector-core/maven-metadata.xml | xmllint --xpath "//metadata/versioning/versions/version[last()]" -
+```
+
+That would return something like to `0.0.1-20230213`. As soon as the version string (here: Feb 13th, 2023) matches the
+current date, we can start the nightly. If the EDC nightly doesn't appear within a set timeframe, we throw an error.
+
+## Notes for becoming OpenSource
+
+- All artifacts (docker images, helm charts, Maven artifacts) should be published to well-known and publicly accessible
+  locations such as MavenCentral, DockerHub, etc. The GitHub Packages repository is only accessible to authenticated
+  users.
```

Comment:
Does this also include our previous releases?

Reply:
Personally I don't see much value in that, provided that the previous releases stay in place.

Reply:
Yes, some opinion.